action.yml

```diff
@@ -32,6 +32,19 @@ inputs:
       for the surest of choices.
     required: false
     default: Changelog
+  versions-dropdownlist-configuration:
+    description:
+      YAML configuration how should the dropdownlist appear
+    required: false
+    default: |
+      config:
+        tooltip:
+          devel: Unreleased development version of Package 
+          main: CRAN Released Package v0.7.0
+        text:
+          devel: devel - unreleased
+          main: main - CRAN Released Package
```

Comment:
If installing the `yaml` R package takes more than a minute, this can be converted to an R-native data structure instead of a YAML.

Reply:
Ok, I will move to R-native structure.